radio/src/targets/common/arm/stm32/usbd_desc.c

```diff
@@ -68,8 +68,12 @@
 #define USBD_VID_PID_CODES                  0x1209    // https://pid.codes
 
 #define USBD_LANGID_STRING                  0x409
-#define USBD_MANUFACTURER_STRING            "OpenTX"
+#define USBD_MANUFACTURER_STRING            "EdgeTX"
```

Comment:
I believe this was changed back to "OpenTX" so that it would work with existing flight sim apps.

Reply:
Ok, I will change it back again